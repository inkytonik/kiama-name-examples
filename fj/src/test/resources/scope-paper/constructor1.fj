// class/constructors.spt:wrong constructor name
class A extends Object {
  B() {
    super();
  }
}
