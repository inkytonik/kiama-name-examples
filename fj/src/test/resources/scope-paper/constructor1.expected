src/test/resources/scope-paper/constructor1.fj:3:3:error: constructor name B is not class name A
  B() {
  ^
